.github/workflows/r.yml

```diff
@@ -72,7 +72,10 @@ jobs:
         ./build-package.sh
     - name: Create test environment
       run: |
-        Rscript -e 'source(".create-test-env.R", echo=TRUE)'
+        pip install -e ../../..
```

Comment:
Can we add a small comment explaining what's located at ../../..?

Reply:
Replaced `../../..` with `git rev-parse --show-toplevel`.